src/Parser/DemonHunter/Havoc/Modules/Items/MoargBionicStabilizers.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import ITEMS from 'common/ITEMS';
+import Analyzer from 'Parser/Core/Analyzer';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import ItemDamageDone from 'Main/ItemDamageDone';
+
+const MOARG_MODIFIER = 0.25;
+
+const MS_BUFFER = 1000;
+
+/**
+* Equip: Throw Glaive deals 25% increased damage for each enemy hit.
+**/
+
+class MoargBionicStabiliziers extends Analyzer {
+	static dependencies = {
+		combatants: Combatants,
+	};
+	rank = 0;
+	bonusDamage = 0;
+	lastCastTimestamp = 0;
+	enemiesHit = 0;
+	damagePreCalc = 0;
+	modifier = 0;
+
+	on_initialized() {
+		this.active = this.combatants.selected.hasWrists(ITEMS.MOARG_BIONIC_STABILIZERS.id);
+	}
+
+	get averageTargetsHit() {
```

Comment:
This isn't used and neither of the variables exists, I assume you forgot about it? :P would be nice to have in the tooltip.